Libraries/LibGUI/ProcessChooser.h

```diff
@@ -38,6 +39,12 @@ class ProcessChooser final : public GUI::Dialog {
 
     pid_t pid() const { return m_pid; }
 
+    bool is_refresh_enabled() { return m_refresh_enabled; }
+    void set_refresh_enabled(bool enabled) { m_refresh_enabled = enabled; }
+
+    unsigned int get_refresh_interval() { return m_refresh_interval; }
+    void set_refresh_interval(unsigned int interval) { m_refresh_interval = interval; }
```

Comment:
Per [coding style](https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md), please omit `int` when using `unsigned` modifier.

Also, please don't use the `get_` prefix for getters, just use bare words (`refresh_interval()` in this case.)